browser/brave_vpn/win/brave_vpn_helper/brave_vpn_helper_utils.cc

```diff
@@ -94,6 +94,11 @@ std::wstring GetBraveVpnHelperServiceName() {
   return name;
 }
 
+std::wstring GetBraveVpnHelperServiceDescription() {
+  // TODO(anyone): Consider adding a description.
+  return std::wstring();
```

Comment:
```suggestion
  return L"Protects Brave VPN against DNS leaks with Smart Multi-Homed Name Resolution when using IKEv2";
```

Reply:
@bsclifton should this be localized, though? If so, I'll create a follow up issue.

Reply:
Changed to the suggested string in 9b86ae3f819

Reply:
Good question - this text should be exposed when person opens up `services.msc`. I don't think we NEED to localize that one as it's not extremely customer facing... but it would definitely be nice to do.